### PR TITLE
Add python client link in documentation

### PR DESCRIPTION
**Pull request by potiuk**
Fixes: https://github.com/apache/airflow-client-python/issues/26

### CODE REVIEW

File: landing-pages/site/content/en/docs/_index.md [hunk 1]
```diff
@@ -182,3 +182,8 @@ Airflow has an official Helm Chart that will help you set up your own Airflow on
         }
     }
 </style>
+
+## Python API Client
```
Comment (mik-laj) [inline landing-pages/site/content/en/docs/_index.md:186]: Title should be clickable for consistency. All other titles are links.

**Review by kaxil — APPROVED** [review]

**Review by mik-laj — APPROVED** [review]

**Comment by msumit** [comment]
@potiuk I think we should add a link to the Go client as well here

**Comment by mik-laj** [comment]
Go client has not been released yet.
https://github.com/apache/airflow-client-go/releases

**Comment by mik-laj** [comment]
> Projects SHALL publish official releases and SHALL NOT publish unreleased materials outside the development community.

https://www.apache.org/legal/release-policy.html

**Comment by potiuk** [comment]
> > Projects SHALL publish official releases and SHALL NOT publish unreleased materials outside the development community.
> 
> https://www.apache.org/legal/release-policy.html

Yep. My thought exactly - we need to release first and we even have separate issue for go client that @mik-laj created (cc: @houqp )

**Pull request merged**